<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lecteurvue</class>
 <widget class="QMainWindow" name="lecteurvue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1618</width>
    <height>979</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1131</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout"/>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ImageDuDiaporama</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnPrecedent">
         <property name="text">
          <string>Précédent</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSuivant">
         <property name="text">
          <string>Suivant</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnLancer">
         <property name="text">
          <string>Lancer diaporama</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnArreter">
         <property name="text">
          <string>Arreter diaporama</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1618</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuParametres">
    <property name="title">
     <string>Paramètres</string>
    </property>
    <addaction name="actionCharger_diaporama"/>
    <addaction name="actionEnlever_diaporama"/>
    <addaction name="actionModifier_vitesse"/>
    <addaction name="actionFiltrer"/>
   </widget>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionA_propos_de"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuParametres"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCharger_diaporama">
   <property name="text">
    <string>Charger diaporama</string>
   </property>
  </action>
  <action name="actionModifier_vitesse">
   <property name="text">
    <string>Vitesse de défilement</string>
   </property>
  </action>
  <action name="actionEnlever_diaporama">
   <property name="text">
    <string>Enlever diaporama</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionA_propos_de">
   <property name="text">
    <string>A propos de...</string>
   </property>
  </action>
  <action name="actionFiltrer">
   <property name="text">
    <string>Filtrer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
